src/pages/AppList.svelte

```diff
@@ -25,9 +25,14 @@
     filteredApps = apps;
   });
 
+  const appOrder = ["firefox_ios", "fenix", "firefox_desktop"];
```

Comment:
I would uppercase this constant (and put it inline in the onMount method, see below)

```suggestion
  const APP_SORT_ORDER = ["firefox_ios", "fenix", "firefox_desktop"];
```